tests/test_standard_tags.py

```diff
@@ -158,26 +158,3 @@ def test_route(self):
             tags["http.route"] = "/sample_rate_route/{i:int}"
 
         interfaces.library.add_span_tag_validation(request=r, tags=tags)
-
-
-@released(
-    dotnet="2.13.0",
-    golang="1.39.0",
-    java="0.107.1",
-    nodejs="3.2.0",
-    php="0.76.0",
-    python=PYTHON_RELEASE_GA_1_1,
-    ruby="?",
-)
-@coverage.basic
-class Test_StandardTagsClientIp(BaseTestCase):
```

Comment:
Keep this class here, default scenario : ASM enabled, env var not set => IP must be reported